cloudify_aws/ec2/resources/ebs.py

```diff
@@ -289,7 +289,7 @@ def delete(ctx, iface, resource_config, **_):
 
     volume_config = ctx.instance.runtime_properties['resource_config']
     if isinstance(volume_config, dict):
-        deleted_params['DryRun'] = volume_config.get('DryRun') or False
+        deleted_params['DryRun'] = volume_config.get('DryRun') or dry_run
```

Comment:
what will happen if `volume_config.get('DryRun')` has a falsey value? I think you should make sure that if `dry_run` is provided, we give preference to this.